### PR TITLE
Consistent DR specs

### PR DESCRIPTION
**Pull request by connorwstein**
jobSpec.contractAddress doesn't exist for DR jobs, removing that from the migration docs

### CODE REVIEW

File: docs/Oracle Jobs/migration-v1-v2.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ observationSource   = """
                       "expiration": $(decode_log.cancelExpiration),
                       "data": $(encode_data)
                   }>]
-    submit       [type=ethtx to="$(jobSpec.contractAddress)" data="$(encode_tx)"]
+    submit       [type=ethtx to="0x613a38AC1659769640aaE063C651F48E0250454C" data="$(encode_tx)"]
 
     decode_log -> decode_cbor -> fetch -> parse -> encode_data -> encode_tx -> submit
```
Comment (dwightjl) [inline docs/Oracle Jobs/migration-v1-v2.md:145]: Can this be `submit_tx` to be consistent with the other examples, or does it affect how this job functions?

File: docs/Oracle Jobs/migration-v1-v2.md [hunk 2]
```diff
@@ -208,7 +208,7 @@ observationSource   = """
                      "expiration": $(decode_log.cancelExpiration),
                      "data": $(encode_data)
                  }>]
-    send_tx     [type=ethtx to="$(jobSpec.contractAddress)" data="$(encode_tx)"]
+    send_tx     [type=ethtx to="0x613a38AC1659769640aaE063C651F48E0250454C" data="$(encode_tx)"]
 
     decode_log -> fetch -> parse -> multiply -> encode_data -> encode_tx -> send_tx
```
Comment (dwightjl) [inline docs/Oracle Jobs/migration-v1-v2.md:213]: Can this also be `submit_tx` for consistency?

**Review by dwightjl — APPROVED** [review]
LGTM. Just a couple of consistency questions.

File: docs/Oracle Jobs/migration-v1-v2.md (hunk 1 above)
Comment (connorwstein) [reply docs/Oracle Jobs/migration-v1-v2.md:145]: doesnt affect the function - will do

**Review by dwightjl — APPROVED** [review]

**Pull request merged**